### ISSUE TITLE
Performance Measurement

### ISSUE THREAD

**Issue by JinLi711** [zeroknowledgediscovery/quasinet#8]
We need to measure the performance of our classifier. 

However, our classifier only accepts categorical features and only predicts categorical outputs. We need to find datasets that meet this requirement, make predictions on the test set, and compare our performance to existing models in sklearn. The performance should be similar.
